functions/api/roles.js

```diff
@@ -46,6 +46,14 @@ const { isAdminOrSuperAdmin } = require('../middleware/auth')
 2.SuperAdmin,Admin,Branch are the roles supported currently.
 */
 router.get('/', isAdminOrSuperAdmin, async (req, res, next) => {
```

Comment:
q: do we need isAdminOrSuperAdmin check for get roles? do we show roles only to admin?

Reply:
Now it's metadata. They will have to read all top level collections. So we need to check only if the user is authenticated.
Also, the Read routes which we have written with different filters doesn't make sense :)